pkg/portal/portal.go

```diff
@@ -256,13 +244,23 @@ func (p *portal) Run(ctx context.Context) error {
 	return s.Serve(tls.NewListener(p.l, config))
 }
 
+func (p *portal) bearerRoutes(r *mux.Router, k *kubeconfig.Kubeconfig) {
```

Comment:
it seems that there is no need to have bearerRoutes func as a method of `*portal` as it does not take any variables from `p *portal` and does not mutate it